client/state/timezones/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import { TIMEZONES_RECEIVE } from 'state/action-types';
+
+import {
+	rawOffsetsSchema,
+	labelsSchema,
+	continentsSchema
+} from './schema';
+
+export const rawOffsetReducer = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { rawOffsets = [] } ) => rawOffsets
+}, rawOffsetsSchema );
+
+export const labelsReducer = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { labels = {} } ) => labels
+}, labelsSchema );
+
+export const byContinentsReducer = createReducer( [], {
+	[ TIMEZONES_RECEIVE ]: ( state, { byContinents } ) => ( byContinents )
+}, continentsSchema );
```

Comment:
The thing is that the var name, which is gotten from the action object (destructing it), already has taken the `byContinent` name. I've added the suffix `Reducer` to avoid the warning.

I could change this ...

```es6
export const byContinents = createReducer( [], {
  [ TIMEZONES_RECEIVE ]: ( state, action ) => ( { byContinents: action.byContinents )
}, continentsSchema );
```